scripts/mods/thaumcraft.zs

```diff
@@ -206,6 +215,13 @@ craft.remake(<thaumadditions:thaumic_lectern>, ["pretty",
   "L": <iceandfire:lectern>,        # Lectern
 });
 
+# Bath salt recipe (this one works)
+mods.thaumcraft.Crucible.removeRecipe(<thaumcraft:bath_salts>);
+recipes.addShapeless("bathsalts",<thaumcraft:bath_salts> * 8,
+[getAnyVisSalt(),getAnyVisSalt(),getAnyVisSalt(),
```

Comment:
Dont need to call `getAnyVisSalt()` 8 times - ingredient could be just copied to each slot